packages/terra-search-field/src/SearchField.scss

```diff
@@ -4,24 +4,32 @@
 
     .input {
       appearance: none;
+      border-bottom-left-radius: var(--terra-search-field-border-bottom-left-radius, 0.25em);
       border-bottom-right-radius: 0;
       border-right: 0;
+      border-top-left-radius: var(--terra-search-field-border-top-left-radius, 0.25em);
       border-top-right-radius: 0;
       flex: 1 1 auto;
-      margin: 0;
+      margin-bottom: var(--terra-search-field-input-margin-bottom, 0);
+      margin-left: var(--terra-search-field-input-margin-left, 0);
+      margin-right: var(--terra-search-field-input-margin-right, 0);
+      margin-top: var(--terra-search-field-input-margin-top, 0);
       min-width: 0;
       outline-offset: -0.143em;  //Adjust the outline offset for search input in Safari.
     }
 
     .button {
-      border: 0.071em solid #dedfe0;
+      border: var(--terra-search-field-button-border, 0.071em solid #dedfe0);
       border-bottom-left-radius: 0;
-      border-bottom-right-radius: 0.25em;
+      border-bottom-right-radius: var(--terra-search-field-border-bottom-right-radius, 0.25em);
```

Comment:
It will be a verbose name, but I think we should consistently scope these variables under the same name.  Similar to my comment above these should match the same prefix. 

`--terra-search-field-button`